src/ResponseMerger/NuxtResponseMerger.php

```diff
@@ -31,11 +31,12 @@ public function mergeResponses(ResponseInterface $backendResponse, ResponseInter
     $page = $frontendResponse->getBody()->getContents();
     $page = preg_replace('/<main role="main"(.*)><\/main>/', '<main role="main"$1>' . $data['content'] . '</main>', $page);
 
-    // Make NUXT hydrate with correct state i.e. pick up the content and render.
-    // Fort that replace the script tag that initialize the __NUXT__ variable.
+    // Append script element before closing body it will add `window.lupus` global object
```

Comment:
please make proper sentences with trailing point. eg.
Add script to define the global `window.lupus` object, which the frontend may use for initializing its state.